### ISSUE TITLE
Create a more "realisitic" config for testing

### ISSUE THREAD

**Issue by unlikelyzero** [yamcs/quickstart#15]
We are hoping to continue to test yamcs/quickstart as a part of our openmct-yamcs build automation. As a part of this work, I'd like to start iterating on the example config as well as start increasing some of the capabilities of the install. I can see where this may start to diverge from the goals of quickstart in it's current form, so I'm totally open to a different path.

Here is our current documentation and approach to automated quickstart testing https://github.com/akhenry/openmct-yamcs/tree/master/tests

**Comment by fqqb** [comment]
True, the original purpose of this quickstart project was to have an example with as little configuration as possible, while still showing core capabilities. Especially replication, is kind-of an advanced subject already.

I need to give it a bit more thought, but I'm thinking that perhaps this repository should have multiple examples rather than just one (in different subdirectories). Then for example, one example could focus fully on OpenMCT. I guess another option is that you do your own thing in one of your repositories.

Note we also have different examples in our main repository (https://github.com/yamcs/yamcs/tree/master/examples), I've never liked that these are separate from the quickstart example as it creates confusion (people thinking they have to fork the yamcs repository).

I'll come back to this in a few days. Lmk if you have more thoughts.
